src/starknet.rs

```diff
@@ -1106,16 +1132,8 @@ pub(crate) mod handler {
             calldata: &ArrayAbi<Felt252Abi>,
             deploy_from_zero: bool,
         ) {
-            let class_hash = Felt::from_bytes_be(&{
-                let mut data = class_hash.0;
-                data.reverse();
-                data
-            });
-            let contract_address_salt = Felt::from_bytes_be(&{
-                let mut data = contract_address_salt.0;
-                data.reverse();
```

Comment:
There are some cases like this one in which the bytes are reversed before creating the felt, wouldn't it affect the result? It doesn't look like the `From<...>` implementation covers these cases, right?

Reply:
Notice that i use from_bytes_le in the from impl, and this snippet uses from_bytes_be, the reverse turns it into big endian, so it is no longer needed, this is because in the past there wasnt a from_bytes_le method so we had to do this

Reply:
Ow right, I didn't catch that one.